### PR TITLE
Update reset-admin-password.md

### PR DESCRIPTION
**Pull request by TwoMoreThings**
I'm suggesting a different way which has been accepted a number of occasions recently on Slack discussions. I think it's less technical and easier/quicker to implement, and safer as it doesn't result in the site being in a vulnerable state during the process.

### CODE REVIEW

**Comment by busrasengul** [comment]
Hey @TwoMoreThings 
Thank you so much for your PR!! 🙌
I remember some of those Slack conversations, and your suggestion definitely looks great. 
I'll be sure one of us reviews this asap!
#h5yr!!

**Comment by ronaldbarendse** [comment]
@TwoMoreThings The currently documented steps (clearing the `Umbraco.Core.ConfigurationStatus` setting, changing the users password to `default` and running the installer) work great, so lets at least make sure this isn't removed from the documentation 😄 

Using the 'forgotten password' feature might indeed be easier to use though, especially if you setup your projects to always use localhost as SMTP-server in your codebase and transform/update this to the correct SMTP-settings when deploying to production/live.

Just to nit-pick: I wouldn't call the SMTP programs _proxies_: they are local/development SMTP servers.

File: Reference/Security/reset-admin-password.md [hunk 1]
```diff
@@ -4,34 +4,74 @@ versionFrom: 8.0.0
...
-That means that in your web.config file the appsettings should look like this:
 
-```csharp
-<appSettings>
-    <add key="Umbraco.Core.ConfigurationStatus" value="" />
-</appSettings>
+### Method A: Using a pickup directory
+
+Open the web.config file at the root of your site.
+locate the section `<mailsettings>` and configure the `deliveryMethod` and specify an existing location for the email to be saved.
```
Comment (sophie-neale) [inline Reference/Security/reset-admin-password.md:36]:
```suggestion
Locate the section `<mailsettings>` and configure the `deliveryMethod` and specify an existing location for the email to be saved.
```

**Review by sophie-neale — COMMENTED** [review]
Suggested a tiny change for a tiny typo 👀 🙂

**Comment by TwoMoreThings** [comment]
> Just to nit-pick: I wouldn't call the SMTP programs _proxies_: they are local/development SMTP servers.

TLDR; I'm happy for that to be changed. 

I think I picked up the 'proxy' term from docs from one of the two programs mentioned. An alternative might be to call it a 'dummy email server', or 'fake email server' or something like that. I specifically didn't want to call it an email server (even a local email server) as that might sound like a security risk or too complex for someone who's a less technical reader, and I wanted it to be clear that it wouldn't actually send any emails. I also wanted to avoid the terms stub or mock as too technical

**Comment by dampee** [comment]
> @TwoMoreThings The currently documented steps (clearing the `Umbraco.Core.ConfigurationStatus` setting, changing the users password to `default` and running the installer) work great, so lets at least make sure this isn't removed from the documentation 😄

As far as I remember, this is the 'official' way of resetting a password. Using the e-mail forgot password is a workaround. It should definitely not be removed from the documentation!

**Comment by sofietoft** [comment]
Hi @TwoMoreThings  👋 

After chatting with the Docs team, I have a suggestion for you:
How about you add the methods you suggest to a separate article? That way we keep the official way of doing this, while offering other ways as well.

How does that sound?

**Comment by sofietoft** [comment]
Hi @TwoMoreThings !

It's been a while since there has been any activity on this one, so I'll be closing it down for now.
If you're still up for it, it would be awesome if you could add your steps here to a new article and make a new PR as well 💪 

Thanks again for helping us keeping the docs up to date! 💪

**Pull request closed without merging**